src/main/java/seedu/duke/utils/Parser.java

```diff
@@ -0,0 +1,682 @@
+package seedu.duke.utils;
```

Comment:
I think normal it is more like normal association so maybe change the dashed line in the UML?